### ISSUE TITLE
Zavera plugin

### ISSUE THREAD

**Issue by TheAlpha12** [jsergio123/script.module.resolveurl#10]
Hello mate can you look over the zavera plugin please keeps skipping links matey not sure if it's the plugin or not as I doubt this has had an update in a while

**Comment by Rhoughton1974** [comment]
Yes please could the plugin for zevera be fixed please. I am eager to be able use it

**Comment by jsergio123** [comment]
Too early to tell what that website is up to now. They seem to redirecting all domains excluding tvad.me to vev.io. Try this fix https://github.com/jsergio123/script.module.resolveurl/commit/e58fcf476bfed57c1db3988802e51c8b515db89e
